src/components/AppHeader/AppHeader.test.tsx

```diff
@@ -8,10 +9,30 @@ describe("AppHeader", () => {
     const name = "My Tasks"
 
     // Act
-    const { getByText } = render(<AppHeader />)
+    const { getByText } = render(
+      <Provider store={store}>
+        <AppHeader />
+      </Provider>,
+    )
     const received = getByText(name)
 
     // Assert
     expect(received).toBeDefined()
   })
+  it("calls to rtk to open the drawer", () => {
```

Comment:
I would lean towards keeping the `it` statements a little more generic like `opens the drawer`, that way if the implementation of how you're opening the drawer ever changes, you won't have to remember to change this too 👍🏽

Reply:
💪